include/libdnf5/rpm/arch.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF5_RPM_ARCH_HPP
+#define LIBDNF5_RPM_ARCH_HPP
+
+#include <string>
+#include <vector>
+
+
+namespace libdnf5::rpm {
+
+std::vector<std::string> get_supported_arches();
```

Comment:
There is somehow related issue - https://github.com/rpm-software-management/dnf5/issues/600